flashinfer_bench/cli/main.py

```
@@ -39,7 +39,7 @@ def best(args: argparse.Namespace):
                 continue
             logger.info(f"Best solution for {definition}:")
             logger.info(f"- Solution: {trace.solution}")
-            logger.info(f"- Speedup:  {trace.evaluation.performance.speedup_factor:.2f}×")
+            logger.info(f"- Speedup vs. ref: {trace.evaluation.performance.speedup_factor:.2f}×")
```

Ruff (0.15.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)